src/main/java/teammates/ui/webapi/output/SearchLinksStudentData.java

```diff
@@ -0,0 +1,75 @@
+package teammates.ui.webapi.output;
```

Comment:
Could be abstracted out into another file, but I felt that this repo has enough classes

Reply:
I think we should put the logic in `CommonSearchLinksData` and make these two classes inherit from it since these are identical.